rdr_service/alembic/versions/e4eedba5965f_.py

```diff
@@ -33,7 +33,7 @@ def upgrade_rdr():
     sa.Column('genomic_report_state', rdr_service.model.utils.Enum(GenomicReportState), nullable=True),
     sa.Column('created', sa.DateTime(), nullable=True),
     sa.Column('modified', sa.DateTime(), nullable=True),
-    sa.Column('module', sa.String(length=80), nullable=True),
+    sa.Column('module', sa.String(length=80), nullable=False),
     sa.ForeignKeyConstraint(['genomic_set_member_id'], ['genomic_set_member.id'], ),
```

Comment:
Is there any existing null value in the table? Do we need to backfill?

Reply:
No there is a migration script in https://github.com/all-of-us/raw-data-repository/pull/2473/files#diff-ff1245f242c85cbffc46b133ebc0c3bce0f1b46f8c75b299f6e041aca778fce5R41-R59, that will go out with `1.97.1` so that will populate all the data accordingly when moved to prod.

Reply:
> No there is a migration script in https://github.com/all-of-us/raw-data-repository/pull/2473/files#diff-ff1245f242c85cbffc46b133ebc0c3bce0f1b46f8c75b299f6e041aca778fce5R41-R59, that will go out with `1.97.1` so that will populate all the data accordingly when moved to prod.

Actually @wangy70 @j-kanuch I need to make another migration file for this since its going out after 1.97.1. Will add that now.

Reply:
https://github.com/all-of-us/raw-data-repository/pull/2488/commits/60e89493aa792462d3426016bb57f25c237fcfc0